doc/git-wta/index.md

```diff
@@ -0,0 +1,15 @@
+# git wta
+
+## worktree add i.e. create a worktree and checkout <commit-ish> into it
```

Comment:
Escape angular brackets. I believe the following suggestion is correct, but please double check:
```suggestion
## worktree add i.e. create a worktree and checkout \<commit-ish> into it
```